### DISCUSSION TITLE
How should i solve this?

### DISCUSSION THREAD

**Discussion by crazyhold** [ScoopInstaller/Scoop#5773] [Help and Questions]
![image](https://github.com/ScoopInstaller/Scoop/assets/102998730/d2f8f508-69f9-4340-90cd-36e543313831)

**Comment by HUMORCE** [comment]
Please leave more information about the Scoop installation, e.g. `scoop config`, git -C "$(scoop prefix scoop)" log -1 ,`git -C "$(scoop prefix scoop)" status`

Cannot reproduce.

```
PowerShell 7.4.0
PS C:\Users\qwerty> scoop install libportable-floorp
Couldn't find manifest for 'libportable-floorp'.
```

**Reply by crazyhold** [reply]
PowerShell 7.4.0
PS C:\Users\qwzpeg> git -C "$(scoop prefix scoop)" log -1
commit f93028001fbe5c78cc41f59e3814d2ac8e595724 (HEAD -> master, tag: v0.3.1, origin/master, origin/HEAD)
Merge: 8aee6f99 c60df9cd
Author: Hsiao-nan Cheung <niheaven@gmail.com>
Date:   Tue Nov 15 10:50:18 2022 +0800

    chore(release): Bump to version 0.3.1 (#5247)
PS C:\Users\qwzpeg> git -C "$(scoop prefix scoop)" status
On branch master
Your branch is up to date with 'origin/master'.

nothing to commit, working tree clean
PS C:\Users\qwzpeg> scoop config

last_update       scoop_repo                              scoop_branch
-----------       ----------                              ------------
23/12/29 03:39:54 https://github.com/ScoopInstaller/Scoop master

**Reply by crazyhold** [reply]
If it can help the bucket I'm tying to install this from is https://github.com/okibcn/ScoopMaster

Also it only happens when i type libportable-floorp
